components/definitions/rewards.d.ts

```diff
@@ -269,10 +269,7 @@ declare namespace Rewards {
     expirationDate: string
   }
 
-  export interface Balance {
-    total: number
-    wallets: Record<string, number>
-  }
+  export type Balance = number | undefined
```

Comment:
`number | undefined` is tricky, since they both can be falsy. Could this be a use case for `Optional<number>`?

Also, it seems like maybe now we can just drop this type alias and use the actual type name?